RELEASE_NOTES.rst

```diff
@@ -1,3 +1,126 @@
+coala-bears 0.9.0
+=================
+
+This release, we had 40 different contributors, pushing almost 200 commits.
+
+To get a quick overview over the current state of all bears, check out
+https://coala.io/languages.
+
+Here are the important changes and additions, of course coala was upgraded to
+0.9 so you can enjoy all it's new features and bugfixes:
+
+**New Bears:**
+
+Python:
+
+- `MypyBear` - Checks python code for typing using Mypy!
+- `BanditBear` - Checks python code for security issues.
+- `PEP8NotebookBear` - Autocorrects formatting for Python code in Jupyter
+  Notebooks.
+- `PySafetyBear` - Checks for known security vulnerabilities of your Python
+  requirements.
+- `PinRequirementsBear` - Checks if python requirements are pinned precisely.
+- `PycodestyleBear` - Checks python codestyle with better error messages and
+  reliability than `PEP8Bear`, however without automatic correction.
+
+Others:
+
+- `QuotesBear` - Checks and corrects your quoting style for any language!
```

Comment:
IMO the new language independent bears should have their own sub-section.  Preferably a proper sub-section with an anchor, so we can link to them, but that is not important now.